src/controller/BookController.ts

```diff
@@ -0,0 +1,114 @@
+import { bookRepository as repository } from '../database/repositories';
+import Book from '../entity/Book';
+
+export default {
+
+    async create(ctx: any) {
+
+        const {
+            uuid,
+            name,
+            isbn,
+            author,
+            releaseDate
+        } = JSON.parse(JSON.stringify(ctx.request.body));
+    
+        const book = repository.create({
+            uuid: uuid,
+            name: name,
+            isbn: isbn,
+            author: author,
+            releaseDate: releaseDate
+        });
+    
+        ctx.body = await repository.save(book);
+    },
+
+    async getAll(ctx: any) {
+        let books: Book[];
+        
```

Comment:
Great work with adding pagination!

Query can be obtained also with `ctx.query` like `ctx.query.take`.